src/ResourceManager/DataLakeStore/Commands.DataLakeStore/ChangeLog.md

```diff
@@ -18,7 +18,7 @@
         - Additional information about change #1
 -->
 ## Upcoming Release
-
+* Update the sdk version of dataplane to 1.1.14. Please refer https://github.com/Azure/azure-data-lake-store-net/blob/master/CHANGELOG.md#version-1114 for SDK fixes.
```

Comment:
No, please summarize these changes here, that is what the changelog is for.

Reply:
Fixed it. Summarized the fixes.